src/WebJobs.Script.WebHost/App_Data/secrets/host.json

```diff
@@ -1,4 +1,4 @@
 ﻿{
-  "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy",
-  "functionKey": "zlnu496ve212kk1p84ncrtdvmtpembduqp25ajjc"
+    "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy,z3dlq50s00cb3q3k11nil7xyt29ebst2n8rtn0t3",
```

Comment:
I don't think this is a breaking change. You look for both 'masterKey' and 'masterKeys', and use whichever one. My take is that it's worth the little extra effort to have a cleaner schema for the long term.

As for array vs dictionary, I'm not sure. I hear Fabio's point, but at the same time, having to come up with 'meaningless' keys could be a burden?